### ISSUE TITLE
dep artisan:migrate doesn't work anymore (did in 3.x now on 4.x)

### ISSUE THREAD

**Issue by HSPDev** [deployphp/deployer#975]
| Q                 | A
| ----------------- | ---
| Issue Type        | Feature Request
| Deployer Version  | 4.x
| Local Machine OS  | N/A
| Remote Machine OS | N/A

### Description 
In the old version 3.x I could just run dep deploy, and then AFTERWARDS manually run dep artisan:migrate and it would "just work" on the latest deployment without issue.

How can I reproduce this in 4.x for one off commands? I have one off tasks specific to each project that would be nice to be able to run with the dep command, and they are all something that you would need to run after deployment at any given time. 

I also don't want to make the artisan:migrate an "after" command, because I want it to have an extra stage of protection from accidents. 

### Output log
```
dep artisan:migrate production -vvv
➤ Executing task artisan:migrate
[production] > cd /var/www/xxx.com && (which php)
[production] < /usr/bin/php
[production] > cd /var/www/xxx.com && (if [ -h /var/www/xxx.com/release ]; then echo 'true'; fi)

                                             
  [RuntimeException]                         
  Release path does not found.               
  Run deploy:release to create new release.  
                                             

Exception trace:
 () at phar:///usr/local/bin/dep/recipe/deploy/release.php:83
 Deployer\{closure}() at n/a:n/a
 call_user_func() at phar:///usr/local/bin/dep/src/Server/Environment.php:126
 Deployer\Server\Environment->get() at phar:///usr/local/bin/dep/src/Server/Environment.php:176
 Deployer\Server\Environment->parseCallback() at n/a:n/a
 preg_replace_callback() at phar:///usr/local/bin/dep/src/Server/Environment.php:162
 Deployer\Server\Environment->parse() at phar:///usr/local/bin/dep/src/functions.php:691
 Deployer\parse() at phar:///usr/local/bin/dep/src/functions.php:286
 Deployer\run() at phar:///usr/local/bin/dep/recipe/laravel.php:55
 Deployer\{closure}() at n/a:n/a
 call_user_func() at phar:///usr/local/bin/dep/src/Task/Task.php:91
 Deployer\Task\Task->run() at phar:///usr/local/bin/dep/src/Executor/SeriesExecutor.php:40
 Deployer\Executor\SeriesExecutor->run() at phar:///usr/local/bin/dep/src/Console/TaskCommand.php:80
 Deployer\Console\TaskCommand->execute() at phar:///usr/local/bin/dep/vendor/symfony/console/Command/Command.php:256
 Symfony\Component\Console\Command\Command->run() at phar:///usr/local/bin/dep/vendor/symfony/console/Application.php:820
 Symfony\Component\Console\Application->doRunCommand() at phar:///usr/local/bin/dep/vendor/symfony/console/Application.php:187
 Symfony\Component\Console\Application->doRun() at phar:///usr/local/bin/dep/vendor/symfony/console/Application.php:118
 Symfony\Component\Console\Application->run() at phar:///usr/local/bin/dep/src/Deployer.php:186
 Deployer\Deployer->run() at phar:///usr/local/bin/dep/bin/dep:110
 require() at /usr/local/bin/dep:4

artisan:migrate [-p|--parallel]
```

**Comment by elfet** [comment]
Yes, this can be fixed when #922 will be done.

**Comment by elfet** [comment]
Fixed.
